src/prefab.ts

```diff
@@ -211,14 +210,24 @@ export class Prefab {
 
   setConfig(rawValues: { [key: string]: any }) {
     this._configs = Config.digest(rawValues);
+    this.loaded = true;
```

Comment:
I moved this inside setConfig so that loaded would get set properly in all the tests.